### PR TITLE
TINY-9679: Changed select element to default to first option in list if none is selected.

### PR DESCRIPTION
**Pull request by HAFRMO**
Related Ticket: 

Description of Changes:
Backporting issue to patch version

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

**Review by lorenzo-pomili — APPROVED** [review]

**Comment by spocke** [comment]
Maybe change the title to reflect the change since it will end up in the changelog as Backporting issue to patch. Maybe use your original title "Changed select element to default to first option in list if none is selected."

**Comment by HAFRMO** [comment]
> Maybe change the title to reflect the change since it will end up in the changelog as Backporting issue to patch. Maybe use your original title "Changed select element to default to first option in list if none is selected."

Very well. It was an attempt to avoid getting a series of change-requests for the issue that'd result in a Fun Time trying to merge them back together, and didn't think about us using the name of PRs as changelog entries.

**Comment by spocke** [comment]
> > Maybe change the title to reflect the change since it will end up in the changelog as Backporting issue to patch. Maybe use your original title "Changed select element to default to first option in list if none is selected."
> 
> Very well. It was an attempt to avoid getting a series of change-requests for the issue that'd result in a Fun Time trying to merge them back together, and didn't think about us using the name of PRs as changelog entries.

I mean changing the PR title that should not conflict when merging them back. I think the branch information is in the merge details so you can see that it's a hotfix merge.

File: modules/alloy/src/test/ts/browser/ui/form/HtmlSelectTest.ts [hunk 1]
```diff
@@ -1,52 +1,91 @@
...
+          options: [
+            { value: 'alpha', text: 'Alpha' },
+            { value: 'beta', text: 'Beta' },
+            { value: 'gamma', text: 'Gamma' }
+          ]
+        })
+      )
+    );
+
+    it('TBA: Should set/get the selected value using the Representing', () => {
```
Comment (vpyshnenko) [inline modules/alloy/src/test/ts/browser/ui/form/HtmlSelectTest.ts:29]: replace TBA with the real ticket number in several places

**Review by vpyshnenko — APPROVED** [review]

File: modules/alloy/src/test/ts/browser/ui/form/HtmlSelectTest.ts (hunk 1 above)
Comment (HAFRMO) [reply modules/alloy/src/test/ts/browser/ui/form/HtmlSelectTest.ts:29]: These are not related to this ticket and are just due to the reformatting of the file.

**Pull request merged**